Dispatcher/UI/Signup - Login/AuthViewController.swift

```diff
@@ -0,0 +1,371 @@
+import UIKit
+
+enum eyeIconStatus {
+    case conseal
+    case reveal
+}
+
+enum SignupLoginPageCurrentView {
+    case signup
+    case login
+}
+
+class AuthViewController: UIViewController, LoadingViewDelegate {
+
+    @IBOutlet weak var loadingView: LoadingView!
+    
+    @IBOutlet weak var logoImageView: UIImageView!
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var wrongEmailLabel: UILabel!
+
+    @IBOutlet weak var passwordTextField: UITextField!
```

Comment:
All these edit text components have the same shape and behavior.
Looks like we should use here a custom view

Reply:
Fixed - created custom view "FormInputView" and delegate
@abenartz